Gordon360/Services/NewsService.cs

```diff
@@ -25,7 +35,6 @@ public NewsService(IUnitOfWork unitOfWork)
         /// controller
         /// </summary>
         /// <param name="newsID">The SNID (id of news item)</param>
-        /// <returns>The news item</returns>
```

Comment:
Why did you remove this comment?

Reply:
That's a great question. I have no idea- I remember having removed the return tag for UploadImage because I had removed all those return strings, and I think I might have mixed this line up with something like that. Putting it back.